aiida/orm/querybuilder.py

```diff
@@ -658,12 +658,15 @@ def append(
         # input now.
         # First of all, let's make sure the specified
         # the class or the type (not both)
-        if cls and entity_type:
+
+        if (cls is not None) and (entity_type is not None):
```

Comment:
```suggestion
        if cls is not None and entity_type is not None:
```
the parentheses are not needed here and for such simple conditionals do not really contribute to clarity either.